src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -593,6 +593,9 @@ void program::pre_optimize_graph(bool is_internal) {
     // check if there exists some layout incompatibilities and add an reorder node if required
     apply_opt_pass<add_required_reorders>();
 
+    // Modify fused post operation to clamp fp16 output before compilation
```

Comment:
It always process gemm and softmax pattern only.
Naming for clamp_fp16_output and comment are too general. At least could you update the comment explicitly shows for clamping gemm and softmax case?

Reply:
Ok, I will add it.